mne/io/kit/kit.py

```diff
@@ -26,12 +26,20 @@
 from ...epochs import BaseEpochs
 from ..constants import FIFF
 from ..meas_info import _empty_info, _read_dig_points, _make_dig_points
-from .constants import KIT
+from .constants import KIT, LEGACY_AMP_PARAMS
 from .coreg import read_mrk
 from ...externals.six import string_types
 from ...event import read_events
 
 
+class UnsupportedKITFormat(Exception):
```

Comment:
To be consistent with the MaxFilter error in FIFF this should probably subclass `ValueError`